src/custom.scss

```diff
@@ -0,0 +1,5 @@
+$bcgov-font-path: "../node_modules/@bcgov/bootstrap-theme/dist/fonts/";
+
+@import "~@bcgov/bootstrap-theme/dist/css/bootstrap-theme.min.css";
+// @import "~@bcgov/bootstrap-theme/dist/scss/common";
+// @import "~@bcgov/bootstrap-theme/dist/scss/bootstrap-theme";
```

Comment:
It would be good to remove comments before committing files.

Reply:
Agree, removing it.